index.js

```diff
@@ -63,7 +60,7 @@ function fastifyPostgres (fastify, options, next) {
   if (options.native) {
     delete options.native
     if (!pg.native) {
-      console.warn('pg-native not installed, can\'t use native option - fallback to pg module')
+      console.warn(`pg-native not installed, can't use native option - fallback to pg module`)
```

Comment:
I prefer to use interpolation _only_ when there is some interpolation going on.

Reply:
Applied double quotes instead, like this there is no need to escape the single quote ^^

Reply:
we use single quotes, so let's keep that escaped.

Reply:
done ^^